lib/ui/src/modules/shortcuts/actions/shortcuts.test.js

```diff
@@ -38,7 +38,7 @@ describe('manager.shortcuts.actions.shortcuts', () => {
 
     test('should warn about deprecated option names', () => {
       const clientStore = new MockClientStore();
-      const spy = jest.spyOn(console, 'warn');
+      const spy = jest.spyOn(console, 'warn').mockImplementation(() => {});
```

Comment:
Is this meant to swallow prop type warnings instead of failing?

Reply:
No, the test makes sure that the warning is called, which is still checked.  By adding the mock implementation, the console.warnings dont get printing in the console for the duration of the specific test

Reply:
Ah, awesome 😀